ext/standard/crypt.c

```diff
@@ -196,7 +196,6 @@ PHPAPI int php_crypt(const char *password, const int pass_len, const char *salt,
 		} else if (
 				salt[0] == '$' &&
 				salt[1] == '2' &&
-				salt[2] >= 'a' && salt[2] <= 'z' &&
 				salt[3] == '$' &&
 				salt[4] >= '0' && salt[4] <= '3' &&
 				salt[5] >= '0' && salt[5] <= '9' &&
```

Comment:
Shouldn't this be removed and handled in `crypt_blowfish.c` as well? Now if you supply a cost of `44` it will fall back to DES rather than erroring in bcrypt.